src/main/java/org/broadinstitute/hellbender/utils/read/GATKRead.java

```diff
@@ -73,6 +73,21 @@ default boolean isEmpty() {
     void setPosition( final Locatable locatable );
 
     /**
+     * @return The actual contig assigned to the read, regardless of unmapped status. Unlike {@link #getContig}, will
+     *         return non-null for unmapped reads that are assigned a nominal position (eg., the position of their mates).
+     *         Useful for sorting reads in standard BAM/SAM file order -- for other uses, clients should use {@link #getContig}
+     */
+    String getAssignedContig();
+
+    /**
+     * @return The actual start position assigned to the read, regardless of unmapped status. Unlike {@link #getStart}, will
```

Comment:
it would perhaps be useful for `GATKRead` to explicitly list methods from `Locatable` and explain how the semantics are interpreted, eg the fact that `getStart` for `unmapped reads that are assigned a nominal position` returns 0. Otherwise, it's left sort of unspecified and that's bad.